CHANGELOG.md

```diff
@@ -3,6 +3,68 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.3.0
+
+_Apr 27, 2023_
+
+We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add new `DigitalClock` desktop [time picking experience](https://mui.com/x/react-date-pickers/digital-clock/) (#7958) @LukasTy
```

Comment:
Could be worth adding a screenshot here

Reply:
What do you think about the added GIF? Will it be good enough? 🤔